### PR TITLE
iFlight fix 2400TX output // add LED cases

### PR DESCRIPTION
**Pull request by XING-IF**
Just fixed the 2400TX output levels and added the iFlight specific LED handling in the code cases

### CODE REVIEW

File: src/lib/LED/devLED.cpp [hunk 1]
```diff
@@ -222,6 +245,11 @@ static int event()
         #endif
     case wifiUpdate:
         #if defined(TARGET_TX)
+                #if defined(TARGET_TX_IFLIGHT_900)|| defined(TARGET_TX_IFLIGHT_2400)
+                    digitalWrite(GPIO_PIN_LED_GREEN, LOW);
+                    digitalWrite(GPIO_PIN_LED_RED, LOW);
+                    return flashLED(GPIO_PIN_LED_BLUE, GPIO_LED_RED_INVERTED, LEDSEQ_WIFI_UPDATE, sizeof(LEDSEQ_WIFI_UPDATE));
```
Comment (cruwaller) [inline src/lib/LED/devLED.cpp:251]: Should be `GPIO_LED_BLUE_INVERTED`.

**Comment by JyeSmith** [comment]
Ive corrected the define to TARGET_TX_IFLIGHT and ordered the logic a little.

@cruwaller please review changes for your comments.

@pkendall64 I have removed the defines in timeout() to allow for any colour LED.

@XING-IF please test.

**Review by cruwaller — APPROVED** [review]

**Review by pkendall64 — APPROVED** [review]

File: src/lib/LED/devLED.cpp [hunk 2]
```diff
@@ -65,6 +65,11 @@ static void initialize()
             pinMode(GPIO_PIN_LED_RED, OUTPUT);
             digitalWrite(GPIO_PIN_LED_RED, LOW ^ GPIO_LED_RED_INVERTED);
         #endif // GPIO_PIN_LED_RED
+        #if defined(TARGET_TX_IFLIGHT)
+            digitalWrite(GPIO_PIN_LED_GREEN, LOW);
+            digitalWrite(GPIO_PIN_LED_RED, HIGH);
+            digitalWrite(GPIO_PIN_LED_BLUE, LOW);
```
Comment (pkendall64) [inline src/lib/LED/devLED.cpp:71]: Shouldn't all these have `^ GPIO_LED_XXX_INVERTED` on them, throughout the file.

Could we also change the `TARGET_TX_IFLIGHT` flag to something more meaningful that could be used by other targets as well. Perhaps we should specify that an RX or TX must specify the LEDs in order RED, GREEN, BLUE and we do different things based on how many LEDs are defined in the target.

Comment (JyeSmith) [reply src/lib/LED/devLED.cpp:71]: They could but are within the TARGET_TX_IFLIGHT define.  So it doesnt really matter.

Comment (brandonrc) [reply src/lib/LED/devLED.cpp:71]: Yea, I do not like the `defined(TARGET_TX_IFLIGHT)`

**Pull request merged**